prefect_census/utils.py

```diff
@@ -0,0 +1,24 @@
+"""Utilities for common interactions with the Census API"""
+from typing import Optional
+
+from httpx import HTTPStatusError
+from orjson import JSONDecodeError
```

Comment:
`httpx` raises the `JSONDecodeError` from the core `json` module
```suggestion
from json import JSONDecodeError
```